core/templates/services/UpgradedServices.ts

```diff
@@ -135,6 +135,8 @@ import { PlaythroughIssueObjectFactory } from
 import { PlaythroughObjectFactory } from
   'domain/statistics/PlaythroughObjectFactory';
 import { PythonProgramTokenizer } from 'classifiers/python-program.tokenizer';
+import { ReadOnlyTopicObjectFactory } from
```

Comment:
Add it below the ReadOnlyStoryNodeObjectFactory. We want to keep all the imports in sorted order.

Reply:
Sure